backend/app/api/ws/chat_namespace.py

```
@@ -1281,6 +1288,47 @@ async def on_chat_retry(self, sid: str, data: dict) -> dict:
+                labels["forceOverrideBotModel"] = "true"
+                if model_type:
+                    labels["forceOverrideBotModelType"] = model_type
+                task.json = task_json
+                flag_modified(task, "json")
+                db.commit()
+                db.refresh(task)
+                logger.info(
+                    f"[WS] chat:retry updated task labels: modelId={model_id}, "
+                    f"modelType={model_type}"
+                )
+            elif use_model_override:
```

Ruff (0.15.10)

[error] Undefined name `use_model_override`

(F821)